docs/service-brokers/gcp-broker-service-classes/bigquery/docs/overview.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Overview
+type: Overview
+---
+
+Google **BigQuery** is a serverless, highly scalable enterprise data warehouse designed to make all your data analysts productive at an unmatched price-performance.
+BigQuery allows you to capture and analyze data in real time using powerful streaming ingestion capability so that your insights are always current.
```

Comment:
```suggestion
BigQuery allows you to capture and analyze data in real time using powerful streaming ingestion capability so that your insights are always up to date.
```

Small thing - maybe `up to date` sounds better.